### ISSUE TITLE
Document port binds and what they are used for

### ISSUE THREAD

**Issue by sparky8251** [jellyfin/jellyfin-docs#3]
I know it at least binds 8096/tcp (HTTP) and 8920/tcp (HTTPS), but we should check if it binds anything else and document those as well.

**Comment by sparky8251** [comment]
Ah! So they could very well be listening but there's no way of telling without digging into the code. All the more reason to investigate and find out.

**Comment by sparky8251** [comment]
7359/udp seems to be part of the DLNA/Chromecast implementation.

If the server implementations, you can see it bind that udp port and this related code:

```
AddMessageResponder("who is EmbyServer?", true, RespondToV2Message);
AddMessageResponder("who is MediaBrowserServer_v2?", false, RespondToV2Message)
```

**Comment by sparky8251** [comment]
[And going by this bit of code](https://github.com/jellyfin/jellyfin/blob/f8a720d3d8adbdb1f092a42e592dae37ba3f25bb/Emby.Server.Implementations/LiveTv/TunerHosts/HdHomerun/HdHomerunUdpStream.cs#L55), the random udp port is for at least a specific bit of LiveTV support (possibly all).

**Comment by sparky8251** [comment]
Looks like the random port is what a network tuner is told to send the LiveTV streams to.

**Comment by JustAMan** [comment]
Closed by #4.
